### DISCUSSION TITLE
Megatinycore problem

### DISCUSSION THREAD

**Discussion by ronproveniers** [SpenceKonde/ATTinyCore#823] [Support / Q & A]
Hello,
My Arduino IDE 2.2.1 has been working fine with compilation of attiny1626.
Since yesterday i get compile errors like "Compilation error: exec: "cmd": executable file not found in %PATH%"???
My machine: DELL desktop 16GB memory, 2TB SSD, Xenon 8 core, WIN10
If i use other official Arduino boards i have no errors
Also with Arduino ide1.8.9 i get the same error message?
I also have a dell laptop and there the tiny_megacore compiles under ide1.8.9 without any problems.
I have deleted/reinstalled Mega-tiny core, reinstalled Arduino ide 2.2.1, no result..
Its with all Spence cores??? that it gives this error.

Your Help is very much appreciated
Best Regards
Ron
rprovo@xs4all.nl

**Comment by hmeijdam** [comment]
You are posting in the wrong core. Attinycore does not support Attiny1626

You mean [Megatinycore](https://github.com/SpenceKonde/megaTinyCore)
That supports the modern Attinies.
Br,
Hans

**Comment by ronproveniers** [comment]
hello Hans,thanks for you reply..this was just an example.the problem i describe is at all boards supported by Spence Konde!!! also at AtTinyCore.i did not see it at any other boardRegardsRonVerstuurd vanaf mijn iPhoneOp 17 nov 2023 om 18:29 heeft Hans ***@***.***> het volgende geschreven:﻿
You are posting in the wrong core. Attinycore does not support Attiny1626
You mean Megatinycore
That supports the modern Attinies.
Br,
Hans

**Comment by hmeijdam** [comment]
Maybe you have [this](https://github.com/SpenceKonde/ATTinyCore/issues/802#issuecomment-1751882383) problem, like a few others.

**Reply by ronproveniers** [reply]
hello everyone
after fiddling with the windows environmental setting( path) it started working again👍😎👍
Ehhh yeh Windos hhee
no idea how it happened
thanks
Ron
